e2e/test.sh

```diff
@@ -17,6 +17,6 @@ printf "\n\n"
 kubectl create ns astro
 kubectl -n astro apply -f /hack/manifests/
 
-kubectl -n astro wait deployment --timeout=60s --for condition=available -l app.kubernetes.io/name=astro,app.kubernetes.io/instance=astro
+kubectl -n astro wait deployment --timeout=60s --for condition=available -l app.kubernetes.io/name=astro
```

Comment:
This is what I had to change to get the tests to work. The deployment didn't have the instance label, that was only applied to the pods the deployment created, therefore this command would never find the resources. Not sure how it worked before 🤷‍♂